database/src/main/postgres/flows/V1.9.1__get_flow_checkpoints.sql

```diff
@@ -58,7 +58,7 @@ $$
 --      id_checkpoint       - id of retrieved checkpoint
 --      checkpoint_name     - name of retrieved checkpoint
 --      measure_name        - measure name associated with a given checkpoint
---      measure_columns     - measure columns associated with a given checkpoint
+--      measured_columns     - measure columns associated with a given checkpoint
```

Comment:
```suggestion
--      measured_columns    - measure columns associated with a given checkpoint
```

just a tiny one, I have OCD and can't help it :(